config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -22,89 +22,49 @@ spec:
       description: SSP is the Schema for the ssps API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
Strange that on my machine `make manifests` generates the text with newlines.
I have `controller-gen` version `v0.3.0`.

Reply:
I think it's fine either way